questiontype.php

```diff
@@ -394,9 +394,15 @@ public function get_available_questions_with_tags($difficulty, $topic, $category
             SELECT tag_instance.itemid
             FROM {tag} tag
                 JOIN {tag_instance} tag_instance ON tag.id = tag_instance.tagid
-                WHERE strcmp(upper(tag_instance.itemtype), \'QUESTION\') = 0 AND strcmp(upper(tag.name), upper("' . $topic . '")) = 0;
+                WHERE strcmp(upper(tag_instance.itemtype), \'QUESTION\') = 0 AND strcmp(upper(tag.name), upper("' . $topic . '")) = 0
```

Comment:
`WHERE` clause should be at the same level of indentation as the `SELECT` clause